.changeset/perfect-bobcats-tickle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@udecode/plate-common': patch
```

Comment:
```suggestion
'@udecode/plate-common': patch
'@udecode/plate-list': patch
```

meaning the changeset tool on github made a mistake and didn't suggest both packages that changed with the PR.

Reply:
Both packages are listed in the changeset, but the changelog would be the same for both. `isSelectionAtBlockEnd` fix is not related to list so I would create another changes to be more explicit.

Will quickly merge and fix that.